src/assets/i18n/fr.json

```diff
@@ -5,38 +5,38 @@
   " minute": "",
   " minutes": "",
   ".": "",
-  "0=Disabled, blank=inherit": "",
+  "0=Disabled, blank=inherit": "0=Désactivé, vide=hérité",
   "2FA": "",
   "<strong>Dataset:</strong> ": "",
-  "<strong>The following { n, plural, one {snapshot} other {# snapshots} } will be deleted. Are you sure you want to proceed?</strong>": "",
+  "<strong>The following { n, plural, one {snapshot} other {# snapshots} } will be deleted. Are you sure you want to proceed?</strong>": "<strong>{ n, plural, one {L} other {Les} } { n, plural, one {'instantané} other {# instantanés} } { n, plural, one {sera} other {seront} } { n, plural, one {supprimé} other {# supprimés} }. Êtes-vous sûr de vouloir continuer ?</strong>",
```

Comment:
Looks like you wanted to do something like this:
```
<strong>{ n, plural, one {L 'instantané sera supprimé} other {Les # instantanés seront supprimés} }. Êtes-vous sûr de vouloir continuer ?</strong>
```

Reply:
oh yes sorry i am not used to ICU Message Editor yet. Thank you that's it I'm correcting ;-)